src/style.css

```diff
@@ -1,3 +1,67 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-style: normal;
 }
+
+.header {
+  display: flex;
+  width: auto;
+  height: 60px;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  margin-left: auto;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__item {
+  margin-right: 20px;
+}
+
+.logo {
+  position: absolute;
+}
+
+.links {
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  line-height: 60px;
+  text-transform: uppercase;
+  display: block;
+}
+
+.links:hover,
+.is-active {
+  color: #00ACDC;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  display: block;
+  width: 37px;
+  height: 4px;
+  border-radius: 8px;
+  top: 56px;
```

Comment:
```suggestion
  bottom: 0;
```

Reply:
Not fixed

Reply:
@anastasiiavorobiova 
I did everything as I should have, but now the blue stripe is in... It’s not included in these 60 pixels and the total height is 64 pixels, I’ve already tried everything, I don’t know how to solve this problem